Style.tsx

```diff
@@ -172,8 +171,78 @@ export const styles = StyleSheet.create({
 
   },
   myProfile: {
-    marginRight: 100,
-  }
+    marginRight: 130,
```

Comment:
margin is too high, try to use flex-start or align-self at the start or justify at the baseline/start